src/helm/proxy/clients/openai_client.py

```diff
@@ -187,8 +192,91 @@ def do_it():
             embedding=embedding,
         )
 
+    @staticmethod
+    def _get_tokenizer_name(tokenizer: str) -> str:
+        return "/".join(tokenizer.split("/")[1:])
+
     def tokenize(self, request: TokenizationRequest) -> TokenizationRequestResult:
-        raise NotImplementedError("Use the HuggingFaceClient to tokenize.")
+        # For reproducibility purposes, we use huggingface/gpt2 as the default tokenizer.
+        # except for gpt-3.5 turbo models and gpt-4 that use the tiktoken library.
+
+        if "openai/" not in request.tokenizer:
+            result = self.tokenizer_client.tokenize(
+                # We're assuming the model uses the GPT-2 tokenizer.
+                TokenizationRequest(request.text, tokenizer="huggingface/gpt2")
+            )
+            return result
```

Comment:
Let's just delete this; `AutoClient` should ensure that non-openai requests don't get sent to this client.

Reply:
Done